db/migrate/20180705230311_smart_proxy_capabilities.rb

```diff
@@ -0,0 +1,17 @@
+class SmartProxyCapabilities < ActiveRecord::Migration[5.1]
+  def up
+    rename_table :features_smart_proxies, :smart_proxy_features
+    change_table :smart_proxy_features, :bulk => true do |t|
+      t.primary_key :id
+      t.text :capabilities, array: true
```

Comment:
Cannot use array, a postgres-only feature, in core. Please refactor this to `serialize :capabilities, Array` approach we already use in core, you can also use JSON/YAML, we use this as well.